styles.css

```diff
@@ -0,0 +1,264 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  //display: flex;
+  //background-color: #ff8c00;

```

Comment:
Please remove out commented code in your codebase. This makes your code unclean and brings confusion to other programmers or you in the future.